vllm/core/block/prefix_caching_block.py

```diff
@@ -7,9 +7,11 @@
                                     get_all_blocks_recursively)
 from vllm.core.block.interfaces import Block, BlockAllocator
 from vllm.core.block.naive_block import NaiveBlock, NaiveBlockAllocator
+from vllm.core.evictor_v2 import EvictionPolicy, Evictor, make_evictor
 
 PrefixHash = int
 BlockId = int
+DEFAULT_LAST_ACCESSED_TIME = -1
 
 
 class PrefixCachingBlockAllocator(BlockAllocator):
```

Comment:
can we update docstring with your improvements?

Reply:
Ok, I would put a docstring here to illustract what this varaible is for